src/cli/organizationsCLIController.ts

```diff
@@ -12,7 +12,7 @@ export type Organization = {
 }
 
 export async function selectOrganizationFromConfig() {
-  const { org: orgFromConfig } = StateManager.getState(KEYS.REPO_CONFIG) || {}
+  const { org: orgFromConfig } = (await getRepoConfig()) || {}
```

Comment:
`getRepoConfig` could handle returning `{}` if the config isn't defined, so that we don't need to handled `undefined` whenever we read the config

Reply:
good point, fixed!